examples/step-44/doc/intro.dox

```diff
@@ -69,6 +69,12 @@ A nice overview of issues pertaining to incompressible elasticity (at small stra
 		Dover.
 </ol>
 
+An example where this three-field formulation is used in a coupled problem is documented in
+<ol>
+	<li> J-P. V. Pelteret, D. Davydov, A. McBride, D. K. Vu, and P. Steinmann (2016),
+		Computational electro- and magneto-elasticity for quasi-incompressible media immersed in free space,
+		<em> International Journal for Numerical Methods in Engineering </em>
```

Comment:
Its in the process of being published, so I’ll just add a DOI number and will update with volume, issue and page numbers when they’re available.

> On 24 May 2016, at 23:26, Wolfgang Bangerth <notifications@github.com> wrote:
> 
> In examples/step-44/doc/intro.dox <https://github.com/dealii/dealii/pull/2620#discussion_r64473630>:
> 
> > @@ -69,6 +69,12 @@ A nice overview of issues pertaining to incompressible elasticity (at small stra
> >  		Dover.
> >  </ol>
> >  
> > +An example where this three-field formulation is used in a coupled problem is documented in
> > +<ol>
> > +	<li> J-P. V. Pelteret, D. Davydov, A. McBride, D. K. Vu, and P. Steinmann (2016),
> > +		Computational electro- and magneto-elasticity for quasi-incompressible media immersed in free space,
> > +		<em> International Journal for Numerical Methods in Engineering </em>
> Can you complete the entry with volume, pages, etc?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub <https://github.com/dealii/dealii/pull/2620/files/a8902e5afe3229dbc3a823d07445459a2d04d9b1#r64473630>